homeassistant/components/rollershutter/wink.py

```diff
@@ -0,0 +1,92 @@
+"""
+Support for Wink Shades.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/rollershutter.wink/
+"""
+import logging
+
+from homeassistant.components.rollershutter import RollershutterDevice
+from homeassistant.const import CONF_ACCESS_TOKEN
+
+REQUIREMENTS = ['python-wink==0.7.7']
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the Wink rollershutter platform."""
+    import pywink
+
+    if discovery_info is None:
+        token = config.get(CONF_ACCESS_TOKEN)
+
+        if token is None:
+            logging.getLogger(__name__).error(
+                "Missing wink access_token. "
+                "Get one at https://winkbearertoken.appspot.com/")
+            return
+
+        pywink.set_bearer_token(token)
+
+    add_devices(WinkRollershutterDevice(shade) for shade in
+                pywink.get_shades())
+
+
+class WinkRollershutterDevice(RollershutterDevice):
+    """Representation of a Wink rollershutter (shades)."""
+
+    def __init__(self, wink):
+        """Initialize the rollershutter."""
+        self.wink = wink
+        self._battery = None
+
+    @property
+    def should_poll(self):
+        """Wink Shades don't track their position."""
+        return False
```

Comment:
It certainly could be `True`.
I've shot out a message to Wink support to see if the position will always be null from the API. If so I'd say this should remain as `False` since polling is just a waste.